src/components/icons/PackageUnlimitedIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ComponentWithAs } from '@chakra-ui/system';
+import { createIcon, IconProps } from '@chakra-ui/react';
+
+export const PackageUnlimitedIcon: ComponentWithAs<'svg', IconProps> =
+  createIcon({
+    displayName: 'Unlimited',
+    viewBox: '0 0 24 24',
+    path: (
+      <>
+        <title>Unlimited package icon</title>
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M12 24C18.0749 24 23 19.0749 23 13C23 6.92507 18.0749 2 12 2C5.92507 2 1 6.92507 1 13C1 19.0749 5.92507 24 12 24ZM12 22.0005C7.02898 22.0005 2.99956 17.971 2.99956 13C2.99956 8.02898 7.02898 3.99956 12 3.99956C16.971 3.99956 21.0005 8.02898 21.0005 13C21.0005 17.971 16.971 22.0005 12 22.0005Z"
+          fill="#333333"
+        />
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M12.001 11.6841C11.509 11.2821 10.811 10.7491 10.129 10.3581C9.455 9.97206 8.779 9.73706 8.263 9.73706C7.397 9.73706 6.567 10.0811 5.956 10.6931C5.345 11.3051 5 12.1351 5 13.0001C5 13.8651 5.344 14.6951 5.955 15.3071C6.567 15.9191 7.397 16.2631 8.263 16.2631C8.779 16.2631 9.455 16.0281 10.129 15.6421C10.811 15.2511 11.509 14.7181 12.001 14.3161C12.494 14.7181 13.191 15.2511 13.873 15.6421C14.546 16.0281 15.222 16.2631 15.737 16.2631C16.603 16.2631 17.433 15.9191 18.044 15.3071C18.655 14.6951 19 13.8651 19 13.0001C19 12.1351 18.656 11.3051 18.045 10.6931C17.433 10.0811 16.603 9.73706 15.737 9.73706C15.222 9.73706 14.546 9.97206 13.873 10.3581C13.191 10.7491 12.494 11.2821 12.001 11.6841ZM10.448 13.0001C10.187 13.2051 9.887 13.4271 9.577 13.6331C9.105 13.9451 8.628 14.2631 8.263 14.2631C7.928 14.2631 7.607 14.1301 7.37 13.8931C7.133 13.6561 7 13.3351 7 13.0001C7 12.6651 7.133 12.3441 7.37 12.1071C7.607 11.8701 7.928 11.7371 8.263 11.7371C8.628 11.7371 9.105 12.0551 9.577 12.3671C9.887 12.5731 10.186 12.7951 10.448 13.0001ZM13.555 13.0001C13.816 12.7951 14.116 12.5731 14.426 12.3671C14.897 12.0551 15.373 11.7371 15.737 11.7371C16.072 11.7371 16.393 11.8701 16.63 12.1071C16.867 12.3441 17 12.6651 17 13.0001C17 13.3351 16.867 13.6561 16.63 13.8931C16.393 14.1301 16.072 14.2631 15.737 14.2631C15.373 14.2631 14.897 13.9451 14.426 13.6331C14.116 13.4271 13.816 13.2051 13.555 13.0001Z"
```

Comment:
There seems to be a lot of unnecessary precision in these coordinates. Can you please try optimizing the svg (and other ones too) by using some online tool, eg. https://svgoptimizer.com/